ios/FluentUI/Vnext/List/List.swift

```diff
@@ -150,6 +150,19 @@ extension MSFListView {
                 }
             })
             .buttonStyle(ListCellButtonStyle(tokens: tokens, layoutType: cell.layoutType))
+            if hasDividers {
+                Divider()
+                    .padding(.leading, (tokens.horizontalCellPadding + tokens.iconSize + tokens.iconInterspace))
```

Comment:
This should consider the icon width and interspace only if it's actually there, right?
Otherwise this will happen with the dividers.

![Screen Shot 2021-01-25 at 11 57 37 AM](https://user-images.githubusercontent.com/68076145/105758798-8624d200-5f04-11eb-9049-94a10d5cc69f.png)